src/copas/copas.lua

```diff
@@ -323,8 +323,12 @@ local _skt_mt = {__index = {
 
                    settimeout = function (self,time)
                                   self.timeout=time
-                                  return
+                                  return true
                                 end,
+
+                   skip = function(self, ...) return self.socket:skip(...) end,
```

Comment:
this skip line must go, otherwise it is good to merge. `skip` is a method of the socket module, eg. `socket.skip`, it is not a method of a socket object